pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -764,9 +763,8 @@ func (configgen *ConfigGeneratorImpl) createGatewayTCPFilterChainOpts(
 					networkFilters: filters,
 				},
 			}
-		} else {
-			log.Warnf("gateway %s:%d listener missed network filter", gatewayName, server.Port)
 		}
+		log.Warnf("gateway %s:%d listener missed network filter", gatewayName, server.Port)
```

Comment:
add a metric for this?

Reply:
Not sure if it is worthy. Same should apply to no vs bound to gateway

```
	virtualServices := push.VirtualServicesForGateway(node.ConfigNamespace, gateway)
	if len(virtualServices) == 0 {
		log.Warnf("no virtual service bound to gateway: %v", gateway)
	}
```